MenuXDGCheck.py

```diff
@@ -25,7 +25,44 @@ def __init__(self):
         # $ echo $XDG_DATA_DIRS/applications
         # /var/lib/menu-xdg:/usr/share
         AbstractCheck.AbstractFilesCheck.__init__(
-            self, "MenuXDGCheck", r"/usr/share/applications/.*\.desktop$")
+            self, "MenuXDGCheck", r'(?:/usr|/etc/opt|/opt/.*)/share/applications/.*\.desktop$')
```

Comment:
Since we're adding directories, I wonder if we should add subdirs (one level deep?) of these too, at least I have some  `/usr/share/applications/{kde,kde4}/*.desktop` on my Fedora 25 box.

Reply:
thats already covered by the regexp, right?

Reply:
Ah, right.